deploy/chart/templates/0000_90_olm_01-prometheus-rule.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: olm-alert-rules
+  namespace: openshift-operator-lifecycle-manager
```

Comment:
Shouldn't the namespace be set as a variable (example: `{{ .Values.namespace }}`), like [other files](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/deploy/chart/templates/0000_90_olm_00-service-monitor.yaml#L6)) do in the templates dir?

Reply:
@awgreene updated the PR. PTAL.